drivers/ma_yya_report/app/models/ma_yya_report/report.rb

```diff
@@ -214,9 +215,10 @@ def describe_filter_as_html
         F1a: a_t[:subsequent_current_living_situations].not_eq([]).and(a_t[:followup_previous_period].eq(false)),
         F1b: a_t[:followup_previous_period].eq(false).
           and(Arel.sql(
-                json_contains(:subsequent_current_living_situations,
-                              [215, 206, 207, 225, 204, 205, 329, 314, 332, 336, 335, 435, 410, 421, 411]),
-              )),
+            # Add note explaining why 302 is excluded? (Transitional housing)
+            json_contains(:subsequent_current_living_situations,
+                          HudUtility2024.permanent_situations(as: :current) + HudUtility2024.temporary_situations(as: :current) + HudUtility2024.institutional_situations(as: :current) - [302]),
+          )),
```

Comment:
Updated this to use the HudUtility helper methods rather than raw numbers. Was `302` excluded purposefully or was that a mistake?